app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,33 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserService {
+
+    private UserRepository repository;
+
+    public UserService(UserRepository repository) {
+        this.repository = repository;
+    }
+
+    public List<User> findAll(){
+        return null;
+    }
+
+    public User save(User user){
+        return null;
+    }
+
+    public User update(Long id , User user){
+        return null;
+    }
+
+    public User delete(Long id){
+        return null;
+    }
```

Comment:
이렇게 작성한 경우 한번에 찾아서 모아 볼 수 있도록 `TODO` 태그 주석을 남겨보세요.
```suggestion
    public User delete(Long id){
        // TODO: 미구현
        return null;
    }
```